### PR TITLE
Routing and Front end documentation

### PR DESCRIPTION
**Pull request by aedanmc**
Contained in this PR are the following changes:

1. A fix for the routing bug (see #151)
2. Initial documentation for contributors to the front-end component codebase.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ In the structure of this repo we have:
 - [Front-end Testing](https://github.com/aedanmc/travelo-hey/blob/main/documentation/instructions/FETESTINGDOC.md)
 - [GCP Set Up](https://github.com/aedanmc/travelo-hey/blob/main/documentation/instructions/GCPDOC.md)
 - [User Guide](https://github.com/aedanmc/travelo-hey/blob/main/documentation/instructions/USERDOC.md)
-
+- [Writing components](https://github.com/aedanmc/travelo-hey/blob/main/documentation/instructions/COMPONENTDOC.md)
```
Comment (C-Christensen) [inline README.md:46]: Small detail: capitalizing "c" in the word components would make it consistent with all the other titles. Other than that LGTM.

File: front-end/src/components/locations-page/LocationPage.js [hunk 2]
```diff
@@ -9,66 +9,12 @@ import SingleReview from '../general/SingleReview';
 
 function LocationPage() {
   // grab place_id aka locationID in SearchPage.js
-  console.log(useParams());
-  const [id] = [useParams()];
-  // const [locations, setLocations] = React.useState([]);
+  const params = useParams();
+  const id = params.place_id;
+  console.log(id);
```
Comment (mik3ycodez) [inline front-end/src/components/locations-page/LocationPage.js:14]: Do we need console.log statement?

Comment (mik3ycodez) [reply front-end/src/components/locations-page/LocationPage.js:14]: LGTM otherwise

**Comment by aedanmc** [comment]
Note that merging this PR introduces bug #158 into the system, but will subsequently be fixed in a future PR after this one merges. This branch has outlived its title; another branch with a name more accurate to the fix associated with #158 should take its place.

**Review by mik3ycodez — APPROVED** [review]
LGTM

**Review by C-Christensen — CHANGES_REQUESTED** [review]
I am working on it. I was able to make some progress. I'll keep the team updated!

**Review by C-Christensen — APPROVED** [review]
The issue is fixed. The routing should be working now and we are good to merge these changes into main.

**Review by matthewdbroom13 — APPROVED** [review]
LGTM!

**Pull request merged**